lib/solarwinds_apm/api/tracing.rb

```diff
@@ -46,13 +46,25 @@ module Tracing
       # * Integer (if integer_response: true)
       #
       def solarwinds_ready?(wait_milliseconds = 3000, integer_response: false)
+        if SolarWindsAPM::OTelConfig[:metrics_processor].nil?
+          # OTelConfig not initialize, use sampler solarwinds_ready from http sampler
+          sampler_solarwinds_ready?(wait_milliseconds)
+        else
+          oboe_solarwinds_ready?(wait_milliseconds, integer_response: integer_response)
+        end
```

Comment:
Is this condition ok if `solarwinds_ready?` gets called when in AWS Lambda environment and doing pure Ruby sampling?